wpiutil/src/main/native/thirdparty/llvm/include/wpi/Casting.h

```diff
@@ -801,6 +801,52 @@ template <class X, class Y>
   return unique_dyn_cast_or_null<X, Y>(Val);
 }
 
+//===----------------------------------------------------------------------===//
+// Isa Predicates
+//===----------------------------------------------------------------------===//
+
+/// These are wrappers over isa* function that allow them to be used in generic
+/// algorithms such as `llvm:all_of`, `wpi::none_of`, etc. This is accomplished
```

Comment:
Should be wpi:: instead of llvm: (this wasn’t caught because the regex looks for ::).